vendor/bat-native-ledger/src/bat/ledger/internal/wallet/balance.cc

```diff
@@ -45,6 +45,11 @@ void Balance::Fetch(ledger::FetchBalanceCallback callback) {
   }
 
   std::string payment_id = ledger_->GetPaymentId();
+  if (payment_id.empty()) {
```

Comment:
This changes fixes an issue where the wallet endpoint can be requested with an empty payment ID path component.